inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/ClientFactoryTest.java

```diff
@@ -37,6 +37,7 @@
 import org.apache.inlong.manager.common.consts.SinkType;
 import org.apache.inlong.manager.common.consts.SourceType;
 import org.apache.inlong.manager.common.enums.ClusterType;
+import org.apache.inlong.manager.common.enums.KafkaOffset;
```

Comment:
The KafkaOffset class is only used by Kafka and is not intended for public use. Therefore, I don't think it's appropriate to place it in the common package.

Reply:
OK